packages/react-composites/src/composites/CallComposite/components/SvgWithWordWrapping.tsx

```diff
@@ -39,7 +39,14 @@ export const SvgWithWordWrapping = (props: {
   }, [width, lineHeightPx, text]);
 
   return (
-    <svg role={role} width={width} height={height + bufferHeightPx} ref={svgRef} xmlns="http://www.w3.org/2000/svg">
+    <svg
+      aria-level={1}
```

Comment:
in the event of the compsite being used alongside other UI on a webpage this could be incorrect.
But also it's our brst guess so if a customer needs it different then perhaps we expose an api at that stage.
lgtm!